hack/debug.sh

```diff
@@ -61,14 +62,25 @@ log_system () {
 }
 
 do_master () {
+    # Find the config file
+    config=$(ps wwaux | grep -v grep | sed -ne 's/.*openshift.*--\(master-\)\?config=\([^ ]*\.yaml\).*/\2/p')
+    if [ -z "$config" ]; then
+      config=$(systemctl show -p ExecStart $aos_master_service | sed -ne 's/.*--\(master-\)\?config=\([^ ]*\).*/\1/p')
+    fi
+    if [ -z "$config" ]; then
+      die "Could not find master-config.yaml from 'ps' or 'systemctl show'"
```

Comment:
If the master invocation doesn't use --config, then just output the "ps" output instead. (Or the systemctl show output I guess for consistency, although it's already present in the debug data anyway...)